core/src/main/java/tc/oc/pgm/listeners/ChatDispatcher.java

```diff
@@ -217,6 +218,12 @@ public void sendReply(Match match, Audience audience, MatchPlayer sender, @Text
     sendDirect(match, sender, receiver.getBukkit(), message);
   }
 
+  private static PersonalizedText formatPrivateMessage(String key) {
+    return new PersonalizedText(
+        new PersonalizedTranslatable(key).getPersonalizedText().color(ChatColor.GRAY).italic(true),
+        new PersonalizedText(" {0}: {1}"));
```

Comment:
I think this should be translatable too.

Reply:
@TheMolkaPL In a previous PR(not sure which of mine) I’m pretty sure someone stated they would rather keep formatting “styles" outside of the translation files. I’m been trying to adhere to that. Do you think this situation would be acceptable to be translatable?

Reply:
In Arabic this should be `TEST :applenick3 [A]` since they read right-to-left.

Reply:
Excuse me, but i'm not sure this makes much sense. Minecraft doesn't support right-to-left to start with, and it's not like changing order of parameters in formatting is the way that you apply RTL